Exercises/2-array.js

```diff
@@ -1,5 +1,14 @@
 'use strict';
 
-const array = () => null;
+class A extends Array {
+  constructor(options = []) {
+    super(options);
+    return Object.assign(i => this[i], {
+      push: val => this.push(val), pop: () => this.pop()
+    });
+  }
+}
+// array is function
+const array = new A();
```

Comment:
You need to create function with name `array`, not instance of class. The purpose of this task is using closure for holding data instead of class fields.

Reply:
Run `npm t` before commit, one test fails.